tap_chargebee/streams/base.py

```diff
@@ -26,6 +26,36 @@ def write_schema(self):
             key_properties=self.KEY_PROPERTIES,
             bookmark_properties=self.BOOKMARK_PROPERTIES)
 
+    def get_abs_path(self, path):
+        return os.path.join(os.path.dirname(os.path.realpath(__file__)), path)
+
+    def load_shared_schema_refs(self):
```

Comment:
Add comments in the functions for better understanding

Reply:
Added